governance.md

```diff
@@ -144,17 +147,17 @@ to its charter, will own the decision. In the case of extended debate
 or deadlock, decisions may be escalated to the Steering Committee,
 which is expected to be uncommon.
 
-The exact processes and guidelines for such cross-project
-communication have yet to be formalized, but when in doubt, use
-kubernetes-dev@googlegroups.com and make an announcement at the
-community meeting.
+The [KEP process] is being developed as a way to facilitate definition, agreement and communication of efforts that cross SIG boundaries.
+SIGs are encouraged to use this process for larger efforts.
+This process is also available for smaller efforts within a SIG.
 
 # Repository guidelines
 
-All repositories under Kubernetes github orgs, such as kubernetes and kubernetes-incubator,
-should follow the procedures outlined in the [incubator document](incubator.md). All code projects
-use the [Apache Licence version 2.0](LICENSE). Documentation repositories should use the
-[Creative Commons License version 4.0](https://git.k8s.io/website/LICENSE).
+All new repositories under Kubernetes github orgs should follow the process outlined in the [kubernetes repository guidelines].
+
+All code projects use the [Apache Licence version 2.0](LICENSE). Documentation repositories should use the [Creative Commons License version 4.0](https://git.k8s.io/website/LICENSE).
```

Comment:
Should we move this to the kubernetes-repositories doc?

There's probably other content from the incubator doc that should be moved there, also.